terraform/datadog/concourse.tf

```diff
@@ -0,0 +1,90 @@
+resource "datadog_monitor" "concourse-load" {
+  name               = "${format("%s concourse load", var.env)}"
+  type               = "query alert"
+  message            = "${format("Concourse load is too high: {{value}}. Check VM health. @govpaas-alerting-%s@digital.cabinet-office.gov.uk", var.aws_account)}"
+  escalation_message = "Concourse load still too high: {{value}}."
+  notify_no_data     = false
+  query              = "${format("avg(last_5m):avg:system.load.5{bosh-job:concourse,bosh-deployment:%s} > 200", var.env)}"
```

Comment:
The load.5 is already an average of 5m, then we average again in concourse. But I don't think this is an issue at all, and it works fine.  I am not sure how to express it differently, neither.